scripts/tablegen/tablegen.py

```diff
@@ -18,6 +18,9 @@
 import linecache
 import string
 import sys
+import wget
```

Comment:
should ideally be alphabetical.  I think once your improvements in this script are done, we can run it through pep8 and make sure it lints well.

Reply:
Fixed :)